metricbeat/module/nats/varz/_meta/fields.yml

```diff
@@ -0,0 +1,77 @@
+- name: varz
+  type: group
+  description: >
+    Contains nats var related metrics
+  fields:
+    - name: server_id
+      type: keyword
+      description: >
+        The server ID
+    - name: now
+      type: date
+      description: >
+        Server time of metric creation
+    - name: uptime
+      type: long
+      description: >
+        The period the server is up
```

Comment:
Can you specify here if this values are seconds or milliseconds. By writing the values, later you can use and Index Pattern in Kibana to do the automatic conversion for you in a dashboard as you can see [here](https://www.elastic.co/guide/en/kibana/current/field-formatters-numeric.html). Check naming conventions when doing it https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html

Reply:
Sure. The values are in seconds. The description will be updated.

Reply:
Thanks!